pyodide_build/buildpkg.py

```diff
@@ -171,7 +171,8 @@ def compile(path: Path, srcpath: Path, pkg: Dict[str, Any], args):
 
     post = pkg.get("build", {}).get("post")
     if post is not None:
-        site_packages_dir = srcpath / "install" / "lib" / "python3.8" / "site-packages"
+        pyfolder = ''.join(["python", os.environ.get("PYMAJOR","3"), ".", os.environ.get("PYMINOR","8")])
+        site_packages_dir = os.path.sep.join(srcpath, "install", "lib", pyfolder, "site-packages" )
```

Comment:
I think you would need
```suggestion
        site_packages_dir = os.path.join(srcpath, "install", "lib", pyfolder, "site-packages" )
```
or pass a list of paths